CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 0.12.2 - 2023-06-28
+
+## What's Changed
+
+### Added
+* Add HTML preset by @alexweininger in [#835](https://github.com/microsoft/vscode-azurestaticwebapps/pull/835)
+* Add actions submenu to workspace view title by @alexweininger in [#838](https://github.com/microsoft/vscode-azurestaticwebapps/pull/838)
```

Comment:
I could see this as being by itself in a  `## Changed` header since this is a pretty significant UI change.

... docs team is going to hate us.